src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -1,4 +1,4 @@
-﻿/*
+/*
```

Comment:
Where is the part of the code that depending on the value of includePayload includes the notification payload or not? I think that important piece is missing in the PR but, again, I could have missed something :)